src/poliastro/ephem.py

```diff
@@ -308,6 +308,27 @@ def from_horizons(
         )
         return cls(coordinates, epochs, plane)
 
+    def from_orbit(cls, orbit, epochs, plane=Planes.EARTH_EQUATOR,):
+        """Return 'Ephem` from an `Orbit` at certain epochs.
+
+        Parameters
+        ----------
+        orbit: ~poliastro.twobody.orbit.Orbit
+            Orbit.
+        epochs: ~astropy.time.Time
+            Epochs to sample the orbit positions.
+        plane: ~poliastro.frames.Planes, optional
+            Fundamental plane of the frame, default to Earth Equator.
+
+        """
+        if epochs.isscalar:
+            epochs = epochs.reshape(1)
+
+        new_orbit = orbit.propagate(epochs)
+        coordinates = new_orbit.sample(epochs.size)
```

Comment:
I don't think `orbit.propagate(epochs)` works. The effect of this `propagate + sample` is essentially using `poliastro.twobody.propagation.propagate` with `epochs`.

Reply:
Thank you @astrojuanlu for your reply! I am slightly confused as to why `orbit.propagate(epochs)` wouldn't work since it would create a new orbit with parameters corresponding to the passed in `epochs` and that `propagate(...)` will find out the `time_of_flight`. 

What I suspect is that `new_orbit.sample(epochs.size)` must return the `coordinates` along with the velocities (as per the constructor for `Ephem`) but my current implementation seems to miss that.

I am sorry if I am missing something quite trivial. I must figure it out. Also, I shall soon work on building the tests as well!

Thank you.

Reply:
> I am slightly confused as to why orbit.propagate(epochs) wouldn't work since it would create a new orbit with parameters corresponding to the passed in epochs

Your assumption is incorrect - an `Orbit` corresponds to a single `epoch`, and can't have an array of `epochs`.

Reply:
Thank you for correcting me! I should instead propagate the `orbit` according to each epoch in `epochs` and then `sample` the corresponding resulting coordinates (in total `epochs.size` coordinates) from all the propagated orbits which can then be used to create the required `Ephem` object, isn't it?

Reply:
Instead of using `Orbit.sample` or `Orbit.propagate`, I think your best bet will be `poliastro.twobody.propagation.propagate`